.circleci/config.yml

```
@@ -41,7 +41,7 @@ orbs:
           no_output_timeout:
             description: Number of seconds to let job run without output before timeout
             type: integer
-            default: 600
+            default: 1800
```

we should restore this timeout to it's original value as the underlying issue previously preventing this from shipping has been resolved.

```suggestion
            default: 600
```